src/SurveyService.Validation/Option/EditOptionRequestValidator.cs

```diff
@@ -0,0 +1,69 @@
+using FluentValidation;
+using FluentValidation.Validators;
+using LT.DigitalOffice.Kernel.Validators;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Option;
+using LT.DigitalOffice.SurveyService.Validation.Option.Interfaces;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.SurveyService.Validation.Option;
+
+public class EditOptionRequestValidator : ExtendedEditRequestValidator<DbOption, EditOptionRequest>,  IEditOptionRequestValidator
```

Comment:
you don't use DbOption in validator. what for is it in arguments? may be remove it and use BaseEditRequestValidator?

Reply:
Fixed